tinygrad/tensor.py

```diff
@@ -170,7 +170,7 @@ def ones(*shape, **kwargs): return Tensor.full(argfix(*shape), 1, **kwargs)
   @staticmethod
   def arange(start, stop=None, step=1, **kwargs):
     if stop is None: stop, start = start, 0
-    return Tensor.full((math.ceil((stop-start)/step),), step, **kwargs).cumsum() + (start - step)
+    return Tensor([i for i in range(start, stop, step)], **kwargs)
```

Comment:
Yea, I can't merge this, it's a CPU copy now. I'll add fixing the N^2 arange as a $1,000 bounty